Mediapipe.Net/Native/UnsafeNativeMethods/Framework/Format/ImageFormat.cs

```diff
@@ -19,7 +19,7 @@ internal unsafe partial class UnsafeNativeMethods : NativeMethods
         [DllImport(MEDIAPIPE_LIBRARY, ExactSpelling = true)]
         public static extern MpReturnCode mp_ImageFrame__ui_i_i_i_Pui8_PF(
             ImageFormat format, int width, int height, int widthStep, byte* pixelData,
-            ImageFrame.Deleter deleter, out void* imageFrame);
+            out void* imageFrame);
```

Comment:
I noticed the deleters had been deleted here. What's the difference now we use a void* instead?

Reply:
> I noticed the deleters had been deleted here. What's the difference now we use a void* instead?

Current situation:

`ImageFrame` is supposed to acquire the ownership of `pixel_data`. But if `pixel_data` is from .NET-managed memory, `ImageFrame` doesn't really own it - .NET gc owns it. .NET runtime may decide to free this memory at any time, before MediaPipe has finished using it, causing a crash. Passing a void deleter doesn't solve this problem by any means.

My updated code:

We copy `pixel_data` from .NET-managed memory in C++ and let MediaPipe take ownership of the copy. This way MediaPipe will not depend on the original `pixel_data`. MediaPipe defaults to using `delete[]` as the deleter when it no longer needs this data, which is the correct behavior as I use `new uint8[]` in the C++ code.

Reply:
Ah, I think this is better than trying to make do with the .NET memory management. We've been having this issues for a while now ourselves too.

Reply:
You never run into problems in your demo code because SeeShark `Frame` happens to be backed by unmanaged native code and needs to be disposed manually. And your `SendFrame` implementation waits until `outPacket` is returned - which is after MediaPipe is done with the frame - before disposing the frame.

But it's not the case in my project, which is why I spotted this problem.